crates/esplora/src/async_ext.rs

```diff
@@ -12,6 +12,12 @@ use futures::stream::{FuturesOrdered, TryStreamExt};
 
 use crate::map_confirmation_time;
 
+/// Trait to extend [`esplora_client::AsyncClient`] functionality.
+///
+/// This is the async version of [`EsploraExt`]. Refer to [crate-level documentation] for more.
```

Comment:
I couldn't find any more info on the crate-level documentation. Maybe something more will be added later?

Reply:
Woops you are right. Maybe "refer to crate level documentation for usage" will be better?

Reply:
Fixed?

Reply:
Umm.. It also doesn't have any usage docs too. Can't we just point to the `keychain_tracker_esplora` crate for usage? That's where they will find how to use this trait.

Reply:
How about now?